make reports requirements could not be resolved to an installable set of packages

Tried to run `make` respectively `make test-php-style-fix` I get following error:
Note the I used the recent master, all is rebased.

@phil-davis maybe you can help...

```
make
Building core

Note: You can type 'make help' for more targets

php /usr/local/bin/composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for bamarni/composer-bin-plugin 1.6.0 -> satisfiable by bamarni/composer-bin-plugin[1.6.0].
    - bamarni/composer-bin-plugin 1.6.0 requires composer-plugin-api ^2.0 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
make: *** [Makefile:149: lib/composer/phpunit] Error 2
```

--- a/README.md
+++ b/README.md
@@ -26,7 +26,7 @@ For installing ownCloud, see the official
 ## Development Build Prerequisites
 Note that when doing a local development build, you need to have **Composer v2** installed. If your OS provides a lower version than v2, you can install Composer v2 manually. As an example, which may be valid for other releases/distros too, see [How to install Composer on Ubuntu 22.04 | 20.04 LTS](https://www.how2shout.com/linux/how-to-install-composer-on-ubuntu-22-04-20-04-lts/).
 
-You also must have installed `yarn` and `node`.
+You also must have installed `yarn` and `node` (v14 or higher).
 
 ## Contribution Guidelines
 https://owncloud.com/contribute/

--- a/README.md
+++ b/README.md
@@ -26,6 +26,8 @@ For installing ownCloud, see the official
 ## Development Build Prerequisites
 Note that when doing a local development build, you need to have **Composer v2** installed. If your OS provides a lower version than v2, you can install Composer v2 manually. As an example, which may be valid for other releases/distros too, see [How to install Composer on Ubuntu 22.04 | 20.04 LTS](https://www.how2shout.com/linux/how-to-install-composer-on-ubuntu-22-04-20-04-lts/).
 
+You also must have installed `yarn` and `node`.
+
 ## Contribution Guidelines
 https://owncloud.com/contribute/
 

--- a/README.md
+++ b/README.md
@@ -10,7 +10,7 @@ A personal cloud which runs on your own server.**
 
 ![](https://github.com/owncloud/screenshots/blob/master/files/sidebar_1.png)
 
-## Why is this so awesome?
+## Why Is This so Awesome?
 * :file_folder: **Access your Data** You can store your files, contacts, calendars and more on a server of your choosing.
 * :package: **Sync your Data** You keep your files, contacts, calendars and more synchronized amongst your devices.
 * :arrows_counterclockwise: **Share your Data** You share your data with others, and give them access to your latest photo galleries, your calendar or anything else you want them to see.
@@ -19,8 +19,12 @@ A personal cloud which runs on your own server.**
 * :lock: **Encryption** You can encrypt data in transit with secure https connections. You can enable the encryption app to encrypt data on storage for improved security and privacy.
 * ...
 
-## Installation instructions
-https://doc.owncloud.com/server/latest/admin_manual/installation/
+## Installation Instructions
+For installing ownCloud, see the official
+[ownCloud 10](https://doc.owncloud.com/server/latest/admin_manual/installation/) installation manual.
+
+## Development Build Prerequisites
+Note that when doing a local development build, you need to have **Composer v2** installed. If your OS provides a lower version than v2, you can install Composer v2 manually. As an example, which may be valid for other releases/distros too, see [How to install Composer on Ubuntu 22.04 | 20.04 LTS](https://www.how2shout.com/linux/how-to-install-composer-on-ubuntu-22-04-20-04-lts/).
 
 ## Contribution Guidelines
 https://owncloud.com/contribute/
@@ -28,15 +32,14 @@ https://owncloud.com/contribute/
 ## Support
 Learn about the different ways you can get support for ownCloud: https://owncloud.com/support/
 
-## Get in touch
+## Get in Touch
 * :clipboard: [Forum](https://central.owncloud.org)
 * :hash: [IRC channel](https://web.libera.chat/?channels=#owncloud)
 * :busts_in_silhouette: [Facebook](https://facebook.com/ownclouders)
 * :hatching_chick: [Twitter](https://twitter.com/ownCloud)
 
-## Important notice on translations
+## Important Notice on Translations
 Please submit translations via Transifex:
 https://www.transifex.com/projects/p/owncloud/
 
-For detailed information about translations:
-https://doc.owncloud.com/server/latest/developer_manual/core/translation.html
+See the detailed information about [translations](https://doc.owncloud.com/server/latest/developer_manual/core/translation.html) here.

